lib/global_config.js

```diff
@@ -157,3 +158,44 @@ const rest = {
   HTTPS_PROTOCOL: 'https'
 };
 exports.rest = rest;
+
+const defaultXmlParserConfiguration = {
+  ignoreAttributes: true,
+  alwaysCreateTextNode: false,
+  attributeNamePrefix: '@',
+  attributesGroupName: null
+};
+
+/**
+ * Return default configuration for XmlParser.
+ *
+ * @returns {object}
+ */
+exports.getXmlParserConfiguration = function ()
+{
+  if (xmlParserConfiguration === undefined || xmlParserConfiguration  === null) {
+    return defaultXmlParserConfiguration;
+  }
+  return xmlParserConfiguration;
+}
+
+/**
+ * Set custom xmlParser configuration or use default.
+ * Parametrs that you can override:
+ *   ignoreAttributes - default true,
+ *   alwaysCreateTextNode - default true,
+ *   attributeNamePrefix - default '@',
+ *   attributesGroupName - default null
+ *
+ * @param {object} value
+ */
+exports.setXmlParserConfiguration = function (xmlParamers)
+{
+  const resultConfig = {
+    ignoreAttributes: xmlParamers && xmlParamers.ignoreAttributes !== undefined  && xmlParamers.ignoreAttributes !== null ? xmlParamers.ignoreAttributes : defaultXmlParserConfiguration.ignoreAttributes,
```

Comment:
can we simplify those if/then/else for all parameters using e.g. local function?